### DISCUSSION TITLE
Is there an API to restore deleted projects?

### DISCUSSION THREAD

**Discussion by lyqht** [octokit/octokit.js#2323] [General] [answered]
I saw on GitHub docs that [there is a manual way to restore deleted projects within 90 days](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository). Wondering if there is an API available/planned for this functionality?

**Comment by timrogers** [comment, marked as answer]
Hi @lyqht! Thanks for reaching out. Since this is a feature request, I think your best bet would be to post on the GitHub Community at https://github.com/orgs/community/discussions/.
